extension/package.json

```diff
@@ -819,6 +819,11 @@
             "Open",
             "Add to Workspace"
           ]
+        },
+        "java.gradle.buildServer.enabled": {
+          "type": "boolean",
+          "description": "[Experimental] Use build server to synchronize Gradle project when enabled.",
+          "default": false
```

Comment:
Is it supposed to be auto? Enable by default in preview but disabled in stable?

Reply:
It will be a little bit complex if we introduce `auto` at current stage, because we need to modify the redhat.java extension - that is the client to explain the auto value according to the client kind.